devtools/environment.yml

```diff
@@ -62,6 +62,7 @@ dependencies:
   - ipdb~=0.13.4
   - isort~=5.0
   - jedi~=0.17.2
+  - jinja2~=2.11.3
```

Comment:
This package version specification:
```
packagename~=1.2.3
```
format means "install the most recent version of this package which claims it is compatible with the version I am specifying here."  This usually means that only the last specified number is allowed to change. So in the case of the `jinja2` dependency you've listed here, it specifies `2.11.x` -- so long as  `x>=3`  This is a pretty restrictive specification. Generally speaking, for packages that have a non-zero major version number, we want to allow the minor version number to float, since the convention is that things should only be expected to break backward compatibility when the major version changes, not the minor version.

When packages start with a major version of 0 (like `jedi~=0.17.2`) that means they are still unstable, and are prone to break things even during "minor" version updates. So for them, we want to "pin" both the major and minor versions, and let the "patch" (the 3rd number) float.  This is why `ipdb` and `jedi` have all 3 numbers listed here, and `isort` only has the first two.

So, because `jinja2` is claiming to be stable (major version > 0) we only want to specify the first 2 numbers here:
```
jinja2~=2.11
```
That'll let us get any improved functionality that comes out in `2.12` but not unexpectedly expose us to potentially breaking changes when `3.0` is released.